### PR TITLE
photoClusterContextmenu

### PR DESCRIPTION
**Pull request by wronny**
I like the approach suggested in #293 by @loupdavid to be more flexible in choosing whether to zoom into a photo cluster or to view the whole current photo cluster in photo viewer.

See #293 for some pros and cons :-)

### CODE REVIEW

File: src/photosController.js [hunk 1]
```diff
@@ -238,6 +264,37 @@ PhotosController.prototype = {
...
+        var zoomText = t('maps', 'Zoom in');
+        var res =
+            '<ul layerId="' + layerId + '">' +
+            '   <li>' +
+            '       <button class="icon-play viewphotos">' +
+            '           <span>' + viewText + '</span>' +
+            '       </button>' +
+            '   </li>' +
+            '   <li>' +
+            '       <button class="icon-search zoomin">' +
```
Comment (tacruc) [inline src/photosController.js:290]: shall we use some more unique class or an id? zoomin might occure more often on the page or?
@eneiluj what do you think?

**Comment by tacruc** [comment]
looks good to me and is working.
@jancborchardt could you check the layout?

File: src/photosController.js (hunk 1 above)
Comment (wronny) [reply src/photosController.js]: as suggested I changed button class to be more unique

**Comment by tacruc** [comment]
Cool, Thanks

**Pull request merged**